packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_market.js

```diff
@@ -61,9 +62,33 @@ Blockly.Blocks.trade_definition_market = {
         // market options and Jump Diffusion Indices from
         // submarket options for Synthetic Indices
         // until multipliers are available for DBot
-        const market_options = active_symbols
-            .getMarketDropdownOptions()
-            .filter(option => option[1] !== 'cryptocurrency');
+        const excludedCountries = [
+            'gb',
+            'ie',
+            'pt',
+            'se',
+            'pl',
+            'cy',
+            'nl',
+            'si',
+            'lv',
+            'hu',
+            'cz',
+            'ro',
+            'dk',
+            'fi',
+            'lt',
+            'hr',
+            'ee',
+            'sk',
+            'bg',
+            'at',
+        ];
+        const { is_virtual, residence } = DBotStore.instance.client;
+        let market_options = active_symbols.getMarketDropdownOptions().filter(option => option[1] !== 'cryptocurrency');
```

Comment:
what do you think, is it possible to keep market_option constant at its place like that?
```suggestion
        const market_options = active_symbols
            .getMarketDropdownOptions()
            .filter(option => {
                         if (option[1] === 'cryptocurrency') return false;
                         if (!isForexAvailable && option[1] === 'forex') return false;
                         
                         return true; 
                  };
```